.github/workflows/generate_changelog.yml

```diff
@@ -21,12 +21,16 @@ jobs:
         uses: actions/checkout@v4
       - name: Generate changelog
         id: generate_changelog
-        uses: metcalfc/changelog-generator@v4.3.1
+        uses: metcalfc/changelog-generator@d552ecf3366cf9d6da9fd7dbe425325d3d095aa8 # v4.3.1
         with:
           myToken: ${{ secrets.GITHUB_TOKEN }}
           head-ref: ${{ github.event.inputs.head-ref }}
           base-ref: ${{ github.event.inputs.base-ref }}
       - name: Summary
         run: |
-          echo "# Changelog" >> $GITHUB_STEP_SUMMARY
-          echo "${{ steps.generate_changelog.outputs.changelog }}" >> $GITHUB_STEP_SUMMARY
+          cat <<EOF >> $GITHUB_STEP_SUMMARY
+          # Changelog
+          [Compare changes](${{ format('{0}/{1}/compare/{2}...{3}', github.server_url, github.repository, github.event.inputs.base-ref, github.event.inputs.head-ref) }})
```

Comment:
So the changelog is reflected in action result or somewhere else?

Reply:
Sounds good to me!

Reply:
👍

Reply:
https://github.com/DynamoDS/Dynamo/actions/runs/8508112969